psycopg/psycopg/_pipeline.py

```diff
@@ -234,8 +247,14 @@ async def __aexit__(
         except Exception as exc2:
             # Don't clobber an exception raised in the block with this one
             if exc_val:
-                logger.warning("error ignored exiting %r: %s", self, exc2)
+                logger.warning("error ignored syncing %r: %s", self, exc2)
```

Comment:
The same fix could be applied on `Pipeline.__exit__()` above.

Reply:
Of course! Fixing it. Thank you, good catch.